eogrow/utils/map.py

```diff
@@ -163,7 +168,22 @@ def merge_tiffs(
     if dtype is not None:
         gdalwarp_options += f" -ot {GDAL_DTYPE_SETTINGS[dtype]}"
 
-    command = f"gdalwarp {gdalwarp_options} {' '.join(input_filenames)} {merged_filename}"
+    input_filelist = list(input_filenames)
+    command = f"gdalwarp {gdalwarp_options} {' '.join(input_filelist)} {merged_filename}"
+
+    if len(command) > SH_COMMAND_LIMIT or len(input_filelist) > OPEN_FILES_LIMIT:
+        # generate a text file with
```

Comment:
It might be a good idea to create the vrt file as temporary file that is cleaned up afterwards. Is there any reason why you'd like to have the file persisted?

Reply:
Generally when running using ray in a container, this file is cleaned when the container is killed but I've added code to do that.

Reply:
I actually like that vrt is persisted, and only the temp file that serves as input to build vrt is temporary. This way one can still inspect the vrt.